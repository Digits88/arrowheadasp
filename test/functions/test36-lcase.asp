<!-- Test LCase function.
-->

LCase("HeLlO tHeRe") = <%= LCase("HeLlO tHeRe") %><br>
LCase("this is a test") = <%= LCase("this is a test") %><br>
LCase("THIS IS A TEST") = <%= LCase("THIS IS A TEST") %><br>
IsNull(LCase(NULL)) = <%= IsNull(LCase(NULL)) %><br>
