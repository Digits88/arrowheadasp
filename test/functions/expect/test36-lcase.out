<!-- Test LCase function.
-->

LCase("HeLlO tHeRe") = hello there<br>
LCase("this is a test") = this is a test<br>
LCase("THIS IS A TEST") = this is a test<br>
IsNull(LCase(NULL)) = True<br>
